docs/api-reference/json/conversion-reference.md

```diff
@@ -56,6 +56,52 @@ Whenever the `JSONConverter` component finds the `@@type` field, it looks into t
 like that in the configuration object above. These classes can be layers, views, or other objects,
  provided the classes have been registered.
 
+## React components and using `@@type` (experimental)
+
+Conversion happens by default for react components. For example, when this configuration of classes is passed to a
+ `JSONConverter`–
+
+```js
+import React from 'react';
+
+import TestComponent from '@/components/test';
+
+const configuration = {
+  classes: Object.assign({}, { React },
```

Comment:
`React` seems special, it is not a class to be instantiated. Perhaps there should be a special field in the configuration for it to let the application supply it.

```
  classes: Record<string, ...>;
  React: unknown;
  reactComponents: